src/herbpy/action/stacking.py

```diff
@@ -0,0 +1,63 @@
+import prpy, numpy
+from prpy.action import ActionMethod
+
+@ActionMethod
+def StackCups(robot, table, cup, stack, cups_stacked):
+    """
+    @param robot The robot performing the stacking
+    @param cup The cup that is being stacked
+    @param stack The stack of cups the cup is going to be placed on
+    @param cups_stacked A list of the cups that have been stacked 
+    """
+	#Grasp cup
+    env = robot.GetEnv()
+
+    with env:
+        aabb_cup = cup.ComputeAABB()
+        aabb_stack = stack.ComputeAABB()
+        stack_height = 2*aabb_stack.extents()[2] + (len(cups_stacked)+1)*0.03
+        manip = robot.GetActiveManipulator()
```

Comment:
Please take `manip` as an argument and only use `GetActiveManipulator` as the default argument.